hera_sim/visibilities/simulators.py

```diff
@@ -93,6 +94,14 @@ def __init__(self, obsparams=None, uvdata=None, sky_freqs=None,
              self.beams,
              self.beam_ids) = initialize_uvdata_from_params(obsparams)
 
+            if point_source_pos is None:
+                try:
+                    catalog = initialize_catalog_from_params(obsparams)[0]
+                    point_source_pos = np.array([catalog['ra_j2000'], catalog['dec_j2000']]).T * np.pi/180.
+                    point_source_flux = np.atleast_2d(catalog['flux_density_I'])
+                except KeyError:
+                    pass
```

Comment:
I'm not 100% sure why we're okay with just `pass`ing if the coordinate system is what we expected or missing a key :thinking: but I'm sure there is a good reason to do so.

Reply:
It'll raise a `KeyError` if there is no `catalog` entry in the `obsparams` file. This is fine, because we allow the catalog to be passed directly to the class as well.

Reply:
I think it might be helpful to put a brief comment on line 98, something like `# setup point sources from catalog if provided`, with another comment explaining that the `KeyError` is raised if a catalog is not specified in the `obsparams`. I'm convinced that it's ok to `pass` here, but it took a bit of digging to convince myself of that.